JavaScript Proxy SDK/README.MD

```diff
@@ -0,0 +1,158 @@
+# JavaScript Proxy SDK example
+Create a feature and connect it to client app using the JavaScript Proxy SDK and at the end we play around with some strategies.
```

Comment:
I would probably improve the intro a bit. 
"In this tutorial we will use the Unleash JavaScript SDK to enable feature toggling in our Single Page Application."